### PR TITLE
KB migration: Profile DB queries on cloud

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds a new task topic to the cloud guide for profiling database queries. It migrates this [KB article](https://support.magento.com/hc/en-us/articles/115003775033-How-to-profile-database-queries-on-Cloud) to devdocs.

## Affected DevDocs pages

- (NEW) https://devdocs.magento.com/guides/v2.3/cloud/project/profile-database-queries.html
- (NEW) https://devdocs.magento.com/guides/v2.2/cloud/project/profile-database-queries.html

whatsnew
Added a new topic about [enabling database query profiling](https://devdocs.magento.com/guides/v2.3/cloud/project/profile-database-queries.html).

### CODE REVIEW

File: guides/v2.2/cloud/project/profile-database-queries.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+|       19 | 0.00293179 | SELECT count(*) FROM `mcom_logged_messages_queue` WHERE (status = 'FAILED') AND (direction = 'outgoing') ORDER BY `sent_at` ASC |
+|       20 | 0.02663829 | SELECT * FROM `mcom_logged_messages_queue` WHERE (status = 'FAILED') AND (direction = 'outgoing') ORDER BY `sent_at` ASC        |
+|       21 | 0.00736756 | ALTER TABLE mcom_logged_messages_queue DROP INDEX idx_direction                                                                 |
+|       22 | 0.00406736 | ALTER TABLE mcom_logged_messages_queue DROP INDEX status                                                                        |
+|       23 | 0.00815528 | ALTER TABLE mcom_logged_messages_queue DROP INDEX idx_status                                                                    |
+|       24 | 0.37585821 | SELECT * FROM `mcom_logged_messages_queue` WHERE (status = 'FAILED') AND (direction = 'outgoing') ORDER BY `sent_at` ASC        |
++----------+------------+---------------------------------------------------------------------------------------------------------------------------------+
+```
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/profile-database-queries.md]: Add {: .no-copy} to this block.

File: guides/v2.2/cloud/project/profile-database-queries.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+group: cloud-guide
+title: Profile database queries
+functional_areas:
+  - Cloud
+  - Setup
+  - Services
+---
+
+When making changes to a database, you may find it helpful to run profiling to identify the impact of those changes. Profiling maintains a log of database queries and increases run time precision.
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/profile-database-queries.md]: When "run time" is used as an adjective, shouldn't it be hyphenated?

File: guides/v2.2/cloud/project/profile-database-queries.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+To enable database query profiling:
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/profile-database-queries.md]: This is low importance, but most of cloud uses H4 for a procedure title. We don't have a class in the CSS yet. 😖

File: guides/v2.2/cloud/project/profile-database-queries.md (hunk 2 above)
Comment (jeff-matthews) [reply guides/v2.2/cloud/project/profile-database-queries.md]: yes. i'm getting rusty

File: guides/v2.2/cloud/project/profile-database-queries.md (hunk 3 above)
Comment (jeff-matthews) [reply guides/v2.2/cloud/project/profile-database-queries.md]: i thought i might be able to sneak this past you ;)

File: guides/v2.2/cloud/project/profile-database-queries.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.2/cloud/project/profile-database-queries.md]: will do

**Comment by jeff-matthews** [comment]
Thanks @hguthrie. Pushed updated.

File: guides/v2.2/cloud/project/profile-database-queries.md (hunk 3 above)
Comment (hguthrie) [reply guides/v2.2/cloud/project/profile-database-queries.md]: 🤣

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**